.github/workflows/galaxy.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Build and Publish Collection to Ansible Galaxy
+
+'on':
+  release:
+    types: [ published ]
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: collections/ansible_collections/kewlfft/aur
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout git repo
+        uses: actions/checkout@v2
+        with:
+          path: collections/ansible_collections/kewlfft/aur
+
+      - name: Ensure that `version:` in `galaxy.yml` matches `GITHUB_REF`
+        id: version
+        run: |
+          [ "$(cat galaxy.yml | grep version: | awk '{print $2}')" = $(awk -F '/' '{print substr($3, 2)}' <<< ${GITHUB_REF}) ] || exit 1
+          echo ::set-output name=version::$(awk -F '/' '{print substr($3, 2)}' <<< ${GITHUB_REF})
+
+      - name: "Set up Python 3.9"
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+
+      - name: Generate cache keys
+        id: keys
+        run: |
+          echo ::set-output name=pip_cache_dir::$(pip cache dir)
+          # This ensures that the cache is invalidated after a week
+          echo ::set-output name=date::$(date +%Y_%g)
+
+      - name: Pip Cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.keys.outputs.pip_cache_dir }}
+          key: ansible-pip-${{ steps.keys.outputs.date }}
+
+      - name: Install pip packages
```

Comment:
pip3 is not the preferred method to install packages in Arch, the python packages exist in official repo.

Reply:
Yes, I am aware. This is the Github Actions workflow which runs on the Ubuntu 20.04 runner (Github Actions's linux runners are all based on Ubuntu). See [this line](https://github.com/kewlfft/ansible-aur/pull/58/files/91fd5a65132cfa452ce3180b3913afdbbb929b0a..97851cd3087e1603b90bda755f93c4b9ebead136#diff-b30d9bc30be1cca8cb1590e890f63d959d241b9193864dc94a2acd6bb7fd9188R15).

Thanks!

Reply:
Should we amend this part? I don't want to mess up and install anything with pip3 on my machine.

Reply:
This workflow does not run on or touch your local machine. It runs on [Github's hosted runners}(https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners).

Reply:
[Here](https://docs.github.com/en/actions) is the Github actions documentation.

I prefer using CI/CD, but if you don't, I can remove it all together. Then, you would have to push each release of the collection to Ansible Galaxy manually. To do this, you would update the `version:` tag in galaxy.yml (the collection metadata file) and run the following commands:
``` shell
ansible-galaxy collection build 
ansible-galaxy collection publish ./kewlfft-aur-<version here>.tar.gz
```